app/views/shared/_back_to_reverb.html.erb

```diff
@@ -0,0 +1,8 @@
+<a
+  id="back-to-reverb-link"
+  style="display: none;"
+  class="toolbar-button"
+  data-bind="visible: $root.ui.reverbRetirement.referrerIsReverb(), click: function() {$root.ui.reverbRetirement.returnToReverb('toolbar link')};"
```

Comment:
Just to proactively say 'why' here - the click event is wrapped in an anonymous function, as this prevents that function from being executed on page load.  Without it, the click event fires as soon as the page is generated, which is not desirable.